listenbrainz/webserver/views/test/test_user.py

```diff
@@ -43,6 +43,39 @@ def tearDown(self):
         ServerTestCase.tearDown(self)
         DatabaseTestCase.tearDown(self)
 
+    def test_user_redirects(self):
+        url = url_for('user.profile', user_name=self.user.musicbrainz_id)
```

Comment:
would prefer if we just made these `/user/musicbrainz_id/` and `/user/musicbrainz_id` instead of generating the via the url_for function. That way the test is much easier to read.